src/Product.jl

```diff
@@ -0,0 +1,85 @@
+using LinearAlgebra
+using Graphs
+
+struct Product <: AbstractAnsatz
+    tn::Ansatz
+end
+
+Ansatz(tn::Product) = tn.tn
+
+Base.copy(x::Product) = Product(copy(Ansatz(x)))
+Base.similar(x::Product) = Product(similar(Ansatz(x)))
+Base.zero(x::Product) = Product(zero(Ansatz(x)))
+
+function Product(arrays::AbstractArray{<:AbstractVector})
```

Comment:
```suggestion
"""
    Product(arrays::AbstractArray{<:AbstractVector})

Create a [`Product`](@ref) ansatz from a collection of vectors.
"""
function Product(arrays::AbstractArray{<:AbstractVector})
```

Reply:
okay, but gonna change some part of the description